packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -14,7 +19,7 @@ interface Props {
   hideHorizontalTrack?: boolean;
   hideVerticalTrack?: boolean;
   scrollTop?: number;
-  setScrollTop: (event: any) => void;
+  setScrollTop?: (event: any) => void;
```

Comment:
What made you change this prop from mandatory to optional?

Reply:
I noticed some pre-existing HOC did not provide the `setScrollTop` props to this component thus leading to some errors during runtime. I figured it's best to make this optional.

Reply:
think this is correct

Reply:
Great, I can see now that it's in the defaults.

Reply:
Nice!